BuildAndTest.proj

```diff
@@ -2,13 +2,13 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project ToolsVersion="14.0" DefaultTargets="BuildAndTest" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Import Project="$(MSBuildThisFileDirectory)build\Targets\Versions.props" />
+  <Import Project="$(MSBuildThisFileDirectory)build/Targets/Versions.props" />
```

Comment:
Is this change necessary?  If not I'd prefer to stay `\` as the majority of our files have that approach.